src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/HMAC.cs

```diff
@@ -41,6 +41,9 @@ public String HashName
             }
             set
             {
+                if (value == null)
+                    throw new ArgumentNullException(nameof(HashName));
```

Comment:
On netfx this would be (indirectly) ArgumentNullException("name").  But this seems fine.